src/state/reducers/targetReducer.ts

```diff
@@ -0,0 +1,36 @@
+/* eslint-disable no-param-reassign */
+import { createSlice } from '@reduxjs/toolkit';
+import { newTargetCoordinates, newTargetFormData } from '../actions/targetAction';
+import type { RootState } from '../store';
+
+const initialState = {
+  targets: [],
```

Comment:
Redux Toolkit allows for caching and re-fetching with [tags](https://redux-toolkit.js.org/rtk-query/usage/automated-refetching#providing-cache-data), making it easier to use data without managing store. Just saying as a suggestion btw!

Reply:
Looks really useful! I'll take a look